client/state/account-recovery/actions.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
If there aren't any external dependencies, you don't need to include this docblock (same is true for internal dependencies).